src/Catalyst.Core.Lib.Tests/IntegrationTests/P2P/PeerValidationIntegrationTest.cs

```diff
@@ -113,16 +116,16 @@ private async Task Setup()
             await _peerService.StartAsync();
         }
 
-        [Fact(Skip = "false")]
-        [Trait(Traits.TestType, Traits.IntegrationTest)]
-        public async Task PeerChallenge_PeerIdentifiers_Expect_To_Succeed_Valid_IP_Port_PublicKey()
-        {
-            await Setup().ConfigureAwait(false);
-            var valid = await RunPeerChallengeTask(_peerSettings.PublicKey, _peerSettings.BindAddress,
-                _peerSettings.Port).ConfigureAwait(false);
-
-            valid.Should().BeTrue();
-        }
+        // [Fact(Skip = "this wont work as it tries to connect to a real node!! We need to instantiate two sockets here")]
+        // [Trait(Traits.TestType, Traits.IntegrationTest)]
+        // public async Task PeerChallenge_PeerIdentifiers_Expect_To_Succeed_Valid_IP_Port_PublicKey()
+        // {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)